src/model/Computation.js

```diff
@@ -62,6 +62,9 @@ export default class Computation extends Model {
 
 		this.boundsSensitive = true;
 		this.dirty = true;
+
+		// computations don't shuffle
+		this.shuffle = undefined;
```

Comment:
Clearing on the instance is probably a sign the api needs to evolve. If we put on the prototype, then it gets called as noop:
```
constructor (...) {
    ...
}

shuffle () {}
```

But that means it gets called for each change.

So should we add a prototype property for `shouldShuffle` (or the negative `noShuffle`) and test that instead of existence of a `shuffle` method?